examples/FirebaseStream_ESP8266/FirebaseStream_ESP8266.ino

```diff
@@ -42,22 +42,22 @@ void setup() {
   Serial.println();
   Serial.print("connected: ");
   Serial.println(WiFi.localIP());
-  
-  fbase.stream("/bitcoin");
 }
 
 
-void loop() {
-  if (fbase.error()) {
+void loop() {  
+  static FirebaseEventStream stream = fbase.stream("/bitcoin");
```

Comment:
Give it a shot, I thought something was preventing that from working. I think it was you can't construct the event stream until setup() is called which won't happen if you directly initialize the global. 
It isn't really a copyable object with the HTTP connection so you can't create a global then copy this value in.
I think we either need to make it a movable object which is probably the cleanest for the client, or return a unique_ptr<> to the object on the heap. This way is the most standard for c++ but maybe not arduino and looks ungly.